dashboard/config/locales/en.yml

```diff
@@ -490,6 +490,9 @@ en:
     for_parents_and_guardians: "For Parents and Guardians"
     link_a_parent_email: "Link a parent/guardian email address to this account to stay updated on your child’s progress and projects. You will also be able to use this email address for password recovery."
     parent_guardian_email_label: "Parent/guardian email"
+    remove_parent_email_label: "Remove"
+    remove_parent_email_aria_label: "Remove Parent Email"
```

Comment:
I'd suggest that if we feel this level of specificity is necessary for users with screen readers, it's probably also necessary for those without them.

Reply:
aria label removed